scss/_carousel.scss

```diff
@@ -302,8 +302,41 @@
 }
 
 /* rtl:end:ignore */
-// End mod
 
+.carousel-indicators-wrapper {
```

Comment:
Maybe already asked by previous reviewer but there was no way to keep using `.carousel-indicators` but in combination of another class?
It would allow to always use the same class whatever the version of the carousel we use, and probably to handle `margin-bottom: $carousel-indicators-margin-bottom;` only for the wrapper class.